tools/code_format/check_format_test_helper.py

```diff
@@ -146,6 +146,14 @@ def checkFileExpectingOK(filename):
   return status + fixFileExpectingNoChange(filename)
 
 
+def checkPGVStringValidation(filename):
```

Comment:
Why can't this be treated as `checkUnfixableError` like other checkers?

Reply:
OK. Will do it ASAP

Reply:
code changes done, working on fixing build issues locally.

Reply:
done